src/clicache/integration-test/CacheHelperN.cs

```diff
@@ -2389,7 +2389,7 @@ public static void KillJavaProcesses()
           string commandline = item["CommandLine"].ToString();
 
           Util.Log("processId:{0} name:{1}", item["ProcessId"], item["Name"]);
-          if (commandline.Contains("geode.jar"))
+          if (commandline.Contains("gemfire.jar"))
```

Comment:
I'm curious why this one you're switching geode with gemfire.  What makes this one different?